### PR TITLE
fix: avoidance parameter file

### PR DESCRIPTION
**Pull request by yukkysaito**
Signed-off-by: Yukihiro Saito <yukky.saito@gmail.com>

## Description
fix for https://github.com/autowarefoundation/autoware.universe/pull/2534

https://github.com/autowarefoundation/autoware.universe/pull/2534#issuecomment-1364616464

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml [hunk 1]
```diff
@@ -23,19 +24,28 @@
...
+
       prepare_time: 2.0                           # [s]
       min_prepare_distance: 1.0                   # [m]
       min_avoidance_distance: 10.0                # [m]
 
       min_nominal_avoidance_speed: 7.0  # [m/s]
       min_sharp_avoidance_speed: 1.0    # [m/s]
 
-      road_shoulder_safety_margin: 0.5 # [m]
+      road_shoulder_safety_margin: 0.0 # [m]
```
Comment (yukkysaito) [inline autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml:48]: @satoshi-ota which one better to use?
original PR don't change this parameter but the value is 0.0
https://github.com/autowarefoundation/autoware.universe/pull/2534/files

Comment (satoshi-ota) [reply autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml:48]: I think 0.5 is better as default.

**Comment by satoshi-ota** [comment]
@yukkysaito Thanks for your PR.
Sorry, I forgot create PR for awf/launch.

**Review by satoshi-ota — APPROVED** [review]

File: autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml (hunk 1 above)
Comment (yukkysaito) [reply autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml:48]: ok :+1:

Comment (yukkysaito) [inline autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml:48]:
```suggestion
      road_shoulder_safety_margin: 0.5 # [m]
```

Comment (yukkysaito) [reply autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml]: [630a29b](https://github.com/autowarefoundation/autoware_launch/pull/145/commits/630a29bea645fe2d43d462fbec1c8e0e8ff4ce29)

**Comment by yukkysaito** [comment]
@satoshi-ota Thank you for reply. No problem :+1:

**Pull request merged**